src/Pages/CreateProduct/components/ProductForm/index.tsx

```diff
@@ -14,14 +14,21 @@ interface IProps {
   onDone?: () => void;
   onSubmit: (data: ICreateProduct | IProduct) => void;
   onClickBack?: () => void;
+  visible?: boolean;
 }
 
 export const ProductForm = ({
   initialValues,
   onCancel,
   onSubmit,
   onClickBack,
+  visible,
 }: IProps) => {
+  const WrapperBtn = styled.div`
+    position: ${!visible ? "fixed" : "none"};
+    bottom: ${!visible ? "30px" : "0"};
+    right: ${!visible ? "50px" : "0"};
+  `;
```

Comment:
maybe extract it outside & pass prop , something like this: 

interface WrapperBtnProps {
  visible?: boolean;
}

const WrapperBtn = styled.div<WrapperBtnProps>`
  position: ${({visible}) => (visible ? "fixed" : "none")};
`;

Reply:
const WrapperBtn = styled.div<WrapperBtnProps>`
  position: ${({visible}) => (visible ? "fixed" : "none")};
`;

Reply:
Fixed